src/search/move_picker.h

```diff
@@ -18,20 +18,23 @@ enum KillerMoveType {
     Second,
 };
 
+enum PromotionOffsets : int32_t {
+    Good = 8000,
+    Bad = -8000,
+};
+
 enum PickerPhase {
     GenerateSyzygyMoves,
     Syzygy,
     GenerateMoves,
     TtMove,
-    GenerateCaptureScores,
-    CaptureGood,
-    PromotionGood,
+    GenerateTacticalScores,
+    TacticalGood,
```

Comment:
The rest of the code base uses "noisy" instead of tactical :)

Reply:
Changed :)